### PR TITLE
renamed max_voltage to more descriptive voltage_divider_factor

### PR DESCRIPTION
**Pull request by jabbenseth**
depends on https://github.com/ipa320/cob_robots/pull/623
sorry about changes in whitespace

### CODE REVIEW

File: cob_phidget_power_state/scripts/power_state_phidget.py [hunk 1]
```diff
@@ -1,23 +1,24 @@
...
+    VOLTAGE_COLLECTION_TIME = 6.0  # sec
 
     def __init__(self):
         self.voltage = None
         self.current = None
         self.last_update = rospy.Time(0)
         self.charging = False
 
-        self.voltage_max = rospy.get_param("~voltage_max", 59.5)
+        self.voltage_divider_factor = rospy.get_param("~voltage_divider_factor", 59.5)
```
Comment (floweisshardt) [inline cob_phidget_power_state/scripts/power_state_phidget.py]: if this is no more a maximum voltage is there a better default value? If not, I would make this a non-optional parameter and thow an error if missing.

**Review by floweisshardt — CHANGES_REQUESTED** [review]

**Review by floweisshardt — APPROVED** [review]

**Comment by fmessmer** [comment]
imho, the descriptiveness of `voltage_divider_factor` isn't much better...division and factor sounds kind of a contradiction in itself...and it's not used within a division...how  about `voltage_scaling_factor....
just my five cents.....I leave this to you guys...

**Comment by jabbenseth** [comment]
The division part comes from the hardware voltage divider. The resulting values are multiplied. But I see your point. What about `voltage_divider_ratio` and changing the values and computation to `1/voltage_divider_factor` to keep it consistent with the general case described by wikipedia: https://en.wikipedia.org/wiki/Voltage_divider
?

**Comment by fmessmer** [comment]
No need to make it more complicated than it is....if everybody is ok, go for it...(add a descriptive comment, if you like)...

**Comment by jabbenseth** [comment]
@ipa-bnm you implemented the node. whats your opinion?

**Review by fmessmer — APPROVED** [review]

**Comment by benmaidel** [comment]
@ipa-jba too late ;)

**Pull request merged**